docs/usage/high-availability/shoot_high_availability_best_practices.md

```diff
@@ -82,6 +82,22 @@ spec:
       scaleDownUtilizationThreshold: 0.5
 ```
 
+In addition to that, it is also possible to configure the CA priority expander by adding priorities to the worker groups like this:
```

Comment:
Could you explain the consequence of having a priority defined for only a few pools, i.e., what's the default prio for pools not having one?